src/nft/hooks/useWalletBalance.ts

```diff
@@ -0,0 +1,22 @@
+import { parseEther } from '@ethersproject/units'
```

Comment:
Is this necessary? It seems to just be a convenience wrapper for existing functions.

If so, can you define a WalletBalance interface for this to return?

Reply:
This convenience function is used by several components which would all need to be refactored if this was removed. I've added the requested interface.